presto-hive/src/main/java/com/facebook/presto/hive/HiveTableLayoutHandle.java

```diff
@@ -259,6 +264,12 @@ public boolean isAppendRowNumberEnabled()
         return appendRowNumberEnabled;
     }
 
+    @JsonProperty
```

Comment:
The build method needs to set the footer stats unreliable otherwise the property will be lost.